src/pages/products/sgid/society/state-fuel-sites.astro

```diff
@@ -26,7 +26,8 @@ export const metadata: IMetadata = {
 
 const page: IPageMetadata = {
   ...metadata,
-  updateHistory: [`December 2023`],
+  application: 'https://experience.arcgis.com/experience/9a38f59e6f914367b18fbbde037028e2',
+  updateHistory: [`AS NEEDED`],
```

Comment:
I would prefer to keep the history instead of changing to as needed.

Reply:
sure but I am the steward and the update history is as needed. Not sure next update I would remember to make the change or that the history was ever maintained or of any significance

Reply:
I understand the refresh cycle is as needed, but that is separate from the update history. We can certainly add the refresh cycle to these pages in the future and that is a good idea. I recommend updating the sgid index to reflect the refresh cycle since it currently says annually.

<img width="856" alt="image" src="https://github.com/agrc/gis.utah.gov/assets/325813/4d6d6799-8f66-469b-86d3-9c24be9dd109">

We all need to do a better job of keeping the update log current and accurate. So please try to remember to update the log and the website when you make changes to any dataset. It all plays into the quality metrics of the data and is important to users of the data.

Reply:
duly noted